lib/gitlab/client/approval_rules.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Gitlab::Client
+  # Defines methods related to project approval rules.
+  # Does not exist in API documentation
```

Comment:
How did you write the code if there are no docs?

Reply:
I didn't write the code but my mates told me that they found them by inspecting the calls from the UI with Chrome 😅